library/blivet.py

```diff
@@ -660,6 +686,7 @@ def run_module():
         volumes=dict(type='list'),
         packages_only=dict(type='bool', required=False, default=False),
         disklabel_type=dict(type='str', required=False, default=None),
+        safe_mode=dict(type='bool', required=False, default=False),
```

Comment:
Shouldn't the default be `True` here for safety? The documentation implies that `True` is the default and it is implemented via the role variables only.

Reply:
Yes, it should. Good catch.